### DISCUSSION TITLE
Using data from my ddbb to train the model.

### DISCUSSION THREAD

**Discussion by jorgecolon413** [microsoft/sample-app-aoai-chatGPT#83] [Q&A]
Is it possible to ingest data from my database (DDBB) using a method other than .pdf, .txt, .md, or .html? The goal is to train a model with my data so that I can ask questions about it using natural language through a chatbot powered by ChatGPT resources. Essentially, I want to query my data using conversational language via a chatbot.

**Comment by sarah-widder** [comment]
Hi @jorgecolon413 thanks for starting this discussion. Currently Azure OpenAI on your data only supports these limited file types, see more information in the documentation: https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/use-your-data#data-formats-and-file-types . Over time we will expand to additional data sources and file types, please stay tuned.
Another option is to finetune the model with your data https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/fine-tuning?pivots=programming-language-studio 
Hope that helps.
